client/state/index.js

```diff
@@ -136,6 +136,7 @@ export function createReduxStore( initialState = {} ) {
 		isBrowser && require( './happychat/middleware.js' ).default(),
 		isBrowser && require( './analytics/middleware.js' ).analyticsMiddleware,
 		isBrowser && require( './data-layer/wpcom-api-middleware.js' ).default,
+		isBrowser && require( './data-layer/extensions-middleware.js' ).default,
```

Comment:
^ @seear, in case you have something there (since Bernie and I are on `sites-list` rotation)